src/components/Navigator/NavigatorCard.vue

```diff
@@ -55,6 +55,7 @@
               :expanded="openNodes[item.uid]"
               :api-change="apiChangesObject[item.path]"
               :isFocused="focusedIndex === index"
+              :lastFocusWasInside="lastFocusWasInside"
```

Comment:
isnt this the same?
```suggestion
              :isFocused="lastFocusWasInside && focusedIndex === index"
```

Reply:
It's not exactly the same because if we set lastFocusWasInside as a condition to focus we will never get to focus in the first place. You can try it out to see what I mean. But I'm open to solve it in another way.